### DISCUSSION TITLE
Option to disable clipboard display message (toast) at bottom of window

### DISCUSSION THREAD

**Discussion by wreedb** [ghostty-org/ghostty#4165] [Feature Requests, Ideas] [answered]
I can't say what the behavior of other platforms is for this, so I'll preface with
version: 1.0.0, backend: OpenGL, runtime: GTK, OS: Linux (Wayland)

When you copy some text in Ghostty, a small toast notification/message appears to confirm that you copied something.
It's a nitpick, but it would be nice if I could disable this; similar to something like the `resize-overlay`.
An idea if anyone feels compelled, perhaps an option to display a visual bell rather than the toast notification.

**Comment by YotamGurfinkel** [comment, marked as answer]
Opened a pull request for the feature :)
https://github.com/ghostty-org/ghostty/pull/4185
